crds/src/lib.rs

```diff
@@ -3,43 +3,15 @@
 //
```

Comment:
I would completly removed this file and move all the content in the go definitions